CMakeLists.txt

```diff
@@ -31,22 +31,17 @@ endif()
 # find dependencies
 include(config.cmake)
 
-# Functiond to read ttk.module files
-if(VTK_FOUND)
-  # required for both wrappers and paraview
-  # Note, may also use it for basecode it we want a base.module file
-  include(CMake/VTKModule.cmake)
-endif()
+# Use folder as configured by the distribution
+include(GNUInstallDirs)
 
 # Base code
 # ---------
 
 include(CMake/CompilerFlags.cmake)
 include(CMake/BaseCode.cmake)
-add_subdirectory(core)
-
 add_library(baseAll INTERFACE)
-add_library(ttkAll INTERFACE)
```

Comment:
wasn't that necessary for the vtk bundle?

Reply:
I tried the examples that use TTK as third party and they works well. The TTKVTKConfig.cmake continue to define the same target as before so this ttkAll should not be required

Reply:
you mean the examples in examples/c++/ and examples/vtk-c++?

Reply:
Exactly

Reply:
alright, thanks!